.gitignore

```diff
@@ -123,15 +123,15 @@ generated/
 #miscell
 .vscode/
 
-!screenshots/README.md
-screenshots/
```

Comment:
Shouldn't these be excluded still? It's only used by integration tests AFAIK

Reply:
Yes!